crawler/db.py

```diff
@@ -259,6 +261,72 @@ def run_mysql_executemany_query(mysql_conn: CMySQLConnection, sql_query: str, va
         mysql_conn.close()
 
 
+def run_mysql_execute_formatted_query(
+    mysql_conn: CMySQLConnection, formatted_sql_query: str, formatting_args: List[str], query_args: List[Any]
+) -> None:
+    """Executes formatted sql query, unwrapping and batching based on number of input arguments
+
+    Arguments:
+        mysql_conn {CMySQLConnection} -- a client used to interact with the database server
+        formatted_sql_query {str} -- the formatted SQL query to run (unwrapped using % workflow)
+        formatting_args {List[str]} -- arguments to batch and unwrap the formatted sql query
+        query_args {List[Any]} -- additional sql query arguments
+    """
+    # fetch the cursor from the DB connection
+    cursor = mysql_conn.cursor()
+
+    try:
+        # executing the query with values
+        num_formatting_args = len(formatting_args)
+
+        # BN. If ROWS_PER_QUERY value is too high, you may get '2006 (HY000): MySQL server has
```

Comment:
Worth changing this comment to match local variable names?

Reply:
Yep done 👍